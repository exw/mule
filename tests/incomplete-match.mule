match Ok {} with
  | Err x -> x
end
