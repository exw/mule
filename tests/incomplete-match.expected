
Desugared:

	
	  match _ with
	  | Err -> fn $0. 
	                    match _ with
	                    | fn x. x
	                    end $0
	  
	  end ((fn x. Ok x) {})
Type error: mismatched type constructors: <empty> and row containing Ok
